woocommerce/woocommerce
# CYS: fix the footer large pattern - use only one navigation block

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #45307 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:


1. Enable the WooCommerce Beta Tester plugin if you don't have it enabled yet.
2. On your Dashboard, head over to Tools > WCA Test Helper > Features and enable the `customize-store`.
3. Visit the `/wp-admin/admin.php?page=wc-admin&path=/customize-store`.
4. Follow the process.
5. Click on "Choose your Footer"
6. Click on the last one.
8. Ensure that the pattern looks like this:
![image](https://github.com/woocommerce/woocommerce/assets/4463174/cdb5d035-a4cc-4391-8154-4bbd4171f59b)





### Changelog entry




<details>

#### Significance




#### Type




#### Message 

CYS: fix the footer large pattern - use only one navigation block

#### Comment 

</details>
